no-useless-rename triggered when there's no rename

**What version of ESLint are you using?**
v2.11.0

**What parser (default, Babel-ESLint, etc.) are you using?**
Default (babel-eslint shows the same issue)

**Please show your full configuration:**
```json
{
  "parserOptions": {
    "ecmaVersion": 6,
    "sourceType": "module"
  },
  "rules": {
    "no-useless-rename": 1
  }
}
```

**What did you do? Please include the actual source code causing the issue.**
```js
import Foo from 'foo';
import {bar as otherBar} from 'bar';
import {fooBar} from 'foo-bar';
import {baz as baz} from 'baz';
```

**What did you expect to happen?**
Just one warning in line 4.

**What actually happened? Please include the actual, raw output from ESLint.**
```
/path-to-the-file/index.js
  3:9  warning  Import fooBar unnecessarily renamed  no-useless-rename
  4:9  warning  Import baz unnecessarily renamed     no-useless-rename

✖ 2 problems (0 errors, 2 warnings)
```

--- a/lib/rules/no-useless-rename.js
+++ b/lib/rules/no-useless-rename.js
@@ -109,7 +109,7 @@ module.exports = {
             }
 
             if (node.imported.name === node.local.name &&
-                    node.imported !== node.local) {
+                    node.imported.range[0] !== node.local.range[0]) {
                 reportError(node, node.imported, node.local, "Import");
             }
         }
@@ -125,7 +125,7 @@ module.exports = {
             }
 
             if (node.local.name === node.exported.name &&
-                    node.local !== node.exported) {
+                    node.local.range[0] !== node.exported.range[0]) {
                 reportError(node, node.local, node.exported, "Export");
             }
 

